client/js/views/alertsMenuView.js

```diff
@@ -129,4 +132,4 @@ AlertsMenuView = GoldstoneBaseView.extend({
             $('.alerts-all').append(self.alertTemplate(alert));
         });
     }
-});
+});
```

Comment:
Please replace the blank line at the end of the file. And set your code editor to automatically do so.